src/app/shared/model/audio-version.model.ts

```diff
@@ -124,9 +127,12 @@ export class AudioVersionModel extends BaseModel {
       case 'Clean':
         data.explicit = 'clean';
         break;
-      default:
+      case '':
```

Comment:
So when story audio version explicit is null (even if set to null by the user because they don't want a parental advisory tag on this story) but the series has explicit or clean, it will always override the story. Unless I'm missing some other indicator here that explicit was not set.